src/replit/ai/modelfarm/model.py

```diff
@@ -143,7 +143,7 @@ def _parse_streaming_response(self, response) -> Iterator[any]:
                     break
             buffer = buffer[start_idx:]
 
-    async def _parse_streaming_aresponse(self, response) -> Iterator[any]:
+    async def _parse_streaming_aresponse(self, response) -> AsyncIterator[Any]:
```

Comment:
Thanks for fixing this return type error. Can you fix the return type of `_parse_streaming_response` above as well?